src/theory/fp/theory_fp.cpp

```diff
@@ -876,7 +879,23 @@ bool TheoryFp::isRegistered(TNode node) {
   return !(d_registeredTerms.find(node) == d_registeredTerms.end());
 }
 
-void TheoryFp::preRegisterTerm(TNode node) {
+void TheoryFp::preRegisterTerm(TNode node)
+{
+#ifdef CVC4_USE_SYMFPU
```

Comment:
Suggestion: I always prefer to use `Configuration::isBuiltWithSymFPU()` if possible to avoid as many preprocessing directives as possible.